includes/KSES.php

```diff
@@ -525,21 +535,61 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 				'srcset'        => true,
 				'srcwidth'      => true,
 			],
+			'svg'                       => [
+				'width'  => true,
+				'height' => true,
+			],
+			'defs'                      => [],
+			'clippath'                  => [
+				'transform'     => true,
+				'clippathunits' => true,
+				'path'          => true,
+			],
+			'path'                      => [
+				'd' => true,
+			],
 		];
 
 		$allowed_tags = array_merge( $allowed_tags, $story_components );
 
-		foreach ( $allowed_tags as &$allowed_tag ) {
-			$allowed_tag['animate-in']          = true;
-			$allowed_tag['animate-in-duration'] = true;
-			$allowed_tag['animate-in-delay']    = true;
-			$allowed_tag['animate-in-after']    = true;
-			$allowed_tag['layout']              = true;
-		}
+		$allowed_tags = array_map( [ $this, 'add_global_attributes' ], $allowed_tags );
 
 		return $allowed_tags;
 	}
 
+	/**
+	 * Helper function to add global attributes to a tag in the allowed HTML list.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @see _wp_add_global_attributes
+	 *
+	 * @param array $value An array of attributes.
+	 * @return array The array of attributes with global attributes added.
+	 */
+	protected function add_global_attributes( $value ) {
+		$global_attributes = [
+			'aria-describedby'    => true,
+			'aria-details'        => true,
+			'aria-label'          => true,
+			'aria-labelledby'     => true,
+			'aria-hidden'         => true,
```

Comment:
^ Curious about adding all these -- are these needed within the post content/was there anything unnecessarily removed in case of using author (etc.) roles?

Reply:
The top ones are the defaults WordPress allows everywhere. But the bottom ones are specific to AMP. There were no issues with ARIA attributes as we‘re not printing those on the frontend